parquet-avro/src/main/java/parquet/avro/AvroReadSupport.java

```diff
@@ -33,9 +40,13 @@
 public class AvroReadSupport<T extends IndexedRecord> extends ReadSupport<T> {
 
   public static String AVRO_REQUESTED_PROJECTION = "parquet.avro.projection";
-  private static final String AVRO_READ_SCHEMA = "parquet.avro.read.schema";
 
-  static final String AVRO_SCHEMA_METADATA_KEY = "avro.schema";
+  public static final String AVRO_READ_SCHEMA = "parquet.avro.read.schema";
+
+  public static final String AVRO_SCHEMA_COMPATIBILITY_CHECK = "parquet.avro.schema.compatibility";
+
+  public static final String AVRO_SCHEMA_METADATA_KEY = "avro.schema";
+
```

Comment:
Nope, no need, I think the original change to the access modifier is now moot -- will change.